tests/prague/eip7692_eof_v1/eip7480_data_section/test_code_validation.py

```diff
@@ -48,10 +55,9 @@
     Container(
         name="max_data_section",
         sections=[
-            Section.Code(
-                code=Op.ADDRESS + Op.POP + Op.STOP,
-            ),
-            Section.Data(data=("1122334455667788" * 8 * 1024)[2:]),
+            Section.Code(code=Op.STOP),
+            # Hits the 49152 bytes limit for the entire container
+            Section.Data(data=(b"12345678" * 6 * 1024)[len(smallest_runtime_subcontainer) :]),
```

Comment:
Multiplying the bytes and then trimming makes a bit hard to deduce what's happening. If we don't need the data to be anything in particular, I think we should do something like:
```suggestion
            Section.Data(data=b"\x00" * (MAX_INITCODE_SIZE - len(smallest_runtime_subcontainer))),
```